jenkins/Jenkinsfile-publish

```diff
@@ -0,0 +1,34 @@
+@Library(['sym-pipeline']) _
+
+import com.symphony.cicd.SymphonyCICDUtils
+
+cicdUtils = new SymphonyCICDUtils()
+
+
+node {
+    withCredentials([
+        file(credentialsId: 'JAVA_SDK_GPG_KEY', variable: 'ORG_GRADLE_PROJECT_KEY'),
+        string(credentialsId: 'JAVA_SDK_GPG_PASSPHRASE', variable: 'ORG_GRADLE_PROJECT_PASSPHRASE'),
+        usernamePassword(credentialsId: 'dev-services', usernameVariable: 'SYM_REPO_USER', passwordVariable: 'SYM_REPO_PASSWORD')
+    ]) {
+    
+      withEnv([
+        "GIT_BRANCH=$PARAM_BUILD_BRANCH",
+        "GIT_ORG=$PARAM_REPO_OWNER", 
+        "GIT_REPO=symphony-api-client-java"
+      ]) {
+
+        stage('Git Checkout') {
+            gitCheckout()
+        }
+
+        stage('Build project') {
+            sh './gradlew clean build'
```

Comment:
`sh './gradlew'` ?

Reply:
while it will be the same to me it is ok for CI to make it more explicit
(adding the default task in the build.gradle is more about making it easy for the developer)